src/app/pages/payment/international-payment/international-payment.component.ts

```diff
@@ -0,0 +1,150 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { InternationalPaymentService } from '../../../services/payment/international-payment.service';
+import { ToastrService } from 'ngx-toastr';
+import { Location } from '@angular/common';
+import {
+  FormGroup,
+  FormBuilder,
+  Validators,
+  ReactiveFormsModule,
+  AbstractControl
+} from '@angular/forms';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { SafeUrl, DomSanitizer } from '@angular/platform-browser';
+import { removeSubscription } from 'src/app/helpers/unsubscribe';
+
+@Component({
+  selector: 'app-payments',
+  templateUrl: './international-payment.component.html',
+  styleUrls: ['./international-payment.component.scss']
+})
+export class InternationalPaymentComponent implements OnInit, OnDestroy {
+  loading: boolean;
+  payload: any;
+  form: FormGroup;
+  formData;
+  subscription: Subscription[] = [];
+  currentUrl: SafeUrl;
+  iframeSrc: SafeUrl;
+  shouldHide = true;
+  expectedMonths: any = [
+    '01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'
+  ];
+
+  expectedYears: any;
+
+  constructor(
+    private internationalPaymentService: InternationalPaymentService,
+    private toastrService: ToastrService,
+    private location: Location,
+    private fb: FormBuilder,
+    private spinner: NgxSpinnerService,
+    private sanitizer: DomSanitizer
+
+  ) { }
+
+  generateYears() {
+    const thisYear = new Date().getFullYear();
+    this.expectedYears = Array.apply(0, Array(14))
+      .map((element, index) => index + thisYear);
+  }
+  ngOnInit() {
+
+    this.generateYears();
+    this.form = this.fb.group({
+      cardno: [
```

Comment:
Variable names are not done well, consider refactoring, use `camelCase`

Reply:
thanks @malep2007 done